cmd/podman/shared/create.go

```diff
@@ -113,6 +113,20 @@ func CreateContainer(ctx context.Context, c *GenericCLIResults, runtime *libpod.
 					if err != nil {
 						return nil, nil, errors.Wrapf(err, "unable to get healthcheck for %s", c.InputArgs[0])
 					}
+
+					// apply defaults if image doesn't override them
+					if healthCheck.Interval == 0 {
```

Comment:
This crashes when image doesn't have healthcheck configured. i.e. `nil` value *is* valid.

Reply:
Maybe all of this needs to be wrapped in a `if healthcheck != nil`?